try {
    data.push(value);
    alert('Action completed!');
    return a + b;
} catch (error) {
    console.error("Error:", error);
}